satellite.toml.default

```diff
@@ -1,6 +1,7 @@
 [Groundstation]
+protocol = "https"
```

Comment:
I don't think this needs to be addressed now, but I don't like how we've got these `.default` files, which are completely seperate to the `Default()` methods in the code. I know why, but it's unintuitive, and should be changed at some point. Probably doesn't need fixing now though.

Reply:
I agree, but yeah, different pr